presto-docs/src/main/sphinx/release/release-0.260.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.260
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug in sql functions that would cause compiler error when session property ``inline_sql_function`` is set to ``false`` and sql function references input in lambda expression.
+* Fix multithread writing for fragment result cache.
+* Add UUID type and operators.
+* Add configuration property ``experimental.aggregation-spill-enabled`` to control aggregate spills explicitly. This can be overridden by ``aggregation_spill_enabled`` session property.
+* Add configuration property ``experimental.order-by-spill-enabled`` to control order by spills explicitly. This can be overridden by ``order_by_spill_enabled`` session property.
+* Add configuration property ``experimental.window-spill-enabled`` to control window spills explicitly. This can be overridden by ``window_spill_enabled`` session property.
+* Add configuration property `fragment-result-cache.max-single-pages-size` to control the max fragement cache file size.
+* Add session property ``query_max_revocable_memory_per_node`` to override existing configuration property ``experimental.max-revocable-memory-per-node``.
+* Add support for Prometheus connector plugin.
```

Comment:
I think this needs to go into the Connectors section.
see https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections

Reply:
This is adding a new connector and not a change in the connector. I am not sure where it needs to go.

Reply:
You need to make a new section `Prometheus Connector Changes` add to that - 

```
* Added Prometheus Connector. See :doc:`connector/prometheus`
```